docs/topics/spiders.rst

```diff
@@ -289,6 +289,31 @@ Spiders receive arguments in their constructors::
             self.start_urls = ['http://www.example.com/categories/%s' % category]
             # ...
 
+To pass more complex arguments like dicts and lists use ``arg-json`` instead.
+Each argument must be JSON-encoded. For example::
+
+    scrapy crawl myspider --arg-json start_urls='["http://www.example.com", "http://dmoz.com"]'
+
+Spider will now start crawling with urls passed via command line.
+
+.. note:: JSON args that contain whitespaces or characters interpreted by Bash like
+double quotes, pipes, etc. should be enclosed with single quotes
+(See example above).
```

Comment:
I'm not sure the syntax is correct - could you please check it locally? 
something like `cd docs; make html; open build/html/index.html`
The text may need to be indented.